src/sentry/static/sentry/app/views/groupDetails/shared/header.jsx

```diff
@@ -91,6 +91,18 @@ const GroupHeader = createReactClass({
     }
   },
 
+  searchTermWithoutQuery() {
```

Comment:
@lynnagara I've pulled the `searchTermWithoutQuery` into a method.

I've done this since I've seen occurrences of `/organizations/default/issues/16/events/?` (without a query string).

I figured it'll be nicer to omit the `?` (question mark) character if there are no query strings.